portal/src/main/webapp/js/src/oncoprint/studyview.js

```diff
@@ -31,9 +115,47 @@ $(document).ready(function() {
 			})();
 			$('#outer_loader_img').hide();
 			$('#oncoprint #everything').show();
+                        
+                        var gene_data = response.toJSON();
+                        //get to process data from gene panel
+                        if(genepanelValues.genepaneldata.length>0)
+                        {
```

Comment:
would you be okay with switching to same-line brackets? I've been using it elsewhere in the file and I think it makes things more concise

``` if (genepanelValues.genepaneldata.length>0) {```